Sources/PhotoCaptureViewController/PhotoCaptureViewController.swift

```diff
@@ -353,6 +356,16 @@ open class PhotoCaptureViewController: UIViewController, PhotoCollectionViewLayo
 
     // MARK: - Actions
 
+    @objc private func handleOrientationChange() {
+        switch UIDevice.current.orientation {
+        case .faceDown, .faceUp, .unknown:
+            ()
```

Comment:
this looks weird.. I guess it's a break

Reply:
Yes, guess it's just a trick to have an executable statement. Copied the old code, but can change it to a `break` instead :)